### PR TITLE
Use GMF fulltext-search web-service for ngeo search example

### LINKED ISSUE #775: Hosted ngeo-search does not work (CORS issue)
http://camptocamp.github.io/ngeo/master/examples/search.html

Could https://geomapfish-demo.camptocamp.net/2.0/wsgi/fulltextsearch also be used in this example?

### PR DESCRIPTION
**Pull request by tsauerwein**
closes #775

### CODE REVIEW

File: examples/search.js [hunk 1]
```diff
@@ -194,6 +194,11 @@ app.module.controller('AppSearchController', app.SearchController);
  * @constructor
  */
 app.MainController = function() {
+  proj4.defs('EPSG:21781',
```
Comment (fredj) [inline examples/search.js]: you can `goog.require('gmf.proj.EPSG21781');`instead

Comment (tsauerwein) [inline examples/search.js]: There are more places that redefine 21781, I will replace them.

But what about using stuff from the gmf module in a pure ngeo example?

**Comment by fredj** [comment]
> But what about using stuff from the gmf module in a pure ngeo example?

Yes, that's a good point ...

File: examples/search.js (hunk 1 above)
Comment (sbrunner) [inline examples/search.js]: Shouldn't that mean that the projections should be in ngeo?

Comment (tsauerwein) [inline examples/search.js]: > Shouldn't that mean that the projections should be in ngeo?

That would make sense!

Comment (fredj) [inline examples/search.js]: > Shouldn't that mean that the projections should be in ngeo?

+1

**Comment by fredj** [comment]
Please merge and open an issue to move `gmf.proj.EPSG21781` to `ngeo`

**Comment by tsauerwein** [comment]
Thanks, for the review. New issue is here: https://github.com/camptocamp/ngeo/issues/789

**Pull request merged**